The release tarball lacks the png and jpg variants of the logos.

diff -Nrup a/pics/Makefile.am b/pics/Makefile.am
--- a/pics/Makefile.am	2006-05-06 09:13:46.000000000 +0200
+++ b/pics/Makefile.am	2026-10-22 17:22:16.156428099 +0200
@@ -1,16 +1,9 @@
 dist_pkgdata_DATA = \
 		    BabyTuX_color.eps \
-		    BabyTuX_color.png \
 		    BabyTuX.eps \
-		    BabyTuX.png \
 		    Debian_color.eps \
-		    Debian_color.png \
 		    Debian.eps \
-		    Debian.png \
 		    Gentoo_color.eps \
-		    Gentoo_color.png \
 		    Gentoo.eps \
-		    Gentoo.png \
 		    penguin.eps \
-		    penguin.jpg \
 		    README.pics
